inc/utils/namespace.php

```diff
@@ -235,6 +235,31 @@ function milliseconds() : int {
 	return (int) microtime( true ) * 1000;
 }
 
+/**
+ * Get past point in time in milliseconds, optionally rounded to the nearest time block.
+ *
+ * @param string $point_in_time strtotime-safe string, eg: '-1 week'
+ * @param integer $round_to Round the result to the nearest time block in seconds, eg: HOUR_IN_SECONDS.
+ *
+ * @return integer|null
+ */
+function time_ago_in_milliseconds( string $point_in_time, int $round_to = 0 ) : ?int {
```

Comment:
Although time ago is our use for this right now it's really a milliseconds wrapper for `strtotime()` which is certainly useful here, I could technically pass in any future date or "+1 week" for example. Maybe the function can be named `str_to_milliseconds` or something along those lines to indicate it's function.

Alternatively if we want to ensure the point in time is in the past then the function should check for that.

Reply:
Yeah, good catch, generalizing it makes more sense.